README.md

```diff
@@ -99,14 +144,32 @@ This macro generates the final sql needed to join the Salesforce formula fields
 ```
 **Args:**
 * `source_table` (required): The table with which you are joining the formula fields.
-* `source_name` (optional, default 'salesforce'): The dbt source containing the table you want to join with formula fields. Must also contain the `fivetran_formula` table.
-* `fields_to_include` (optional, default is none): If a users wishes to only run the formula fields macro for designated fields then they may be applied within this variable. This variable will ensure the model only generates the sql for the designated fields. 
-* `full_statement_version` (optional, default is true): Allows a user to leverage the `fivetran_formula_table` version of the macro which will generate the formula fields via the complete sql statement, rather than individual formulas being generated within the macro.
-* `using_quoted_identifiers` (optional, default is false): For warehouses with case sensitivity enabled this argument **must** be set to `true` in order for the underlying macros within this project to properly compile and execute successfully. 
-* `materialization` (optional, default is `view`): By default the model will be materialized as a view. If you would like to materialize as a table, you can adjust using this argument.
+* `source_name` (optional, default = `'salesforce'`): The dbt source containing the table you want to join with formula fields. Must also contain the `fivetran_formula` table.
```

Comment:
Just for my knowledge, can you give an example for this field to be included and declared? I'm not sure I understand what is meant by joining a table with formula fields

Reply:
yeah so like if a customer [declared](https://github.com/fivetran/dbt_salesforce_formula_utils#source-tables-required) the source with a different name (we recommend salesforce), such as `fivetran_salesforce`, they'd include `source_name='fivetran_salesforce'` as an argument

Reply:
the language is weird tho, lemme update